test/e2e/pod_infra_container_test.go

```diff
@@ -383,12 +383,14 @@ var _ = Describe("Podman pod create", func() {
 		podID := session.OutputToString()
 
 		// verify we can add a host to the infra's /etc/hosts
-		session = podmanTest.Podman([]string{"run", "--pod", podID, "--add-host", "foobar:127.0.0.1", BB, "ping", "-c", "1", "foobar"})
+		// N/B: Using alpine for ping, since BB ping throws
+		//      permission denied error as of Fedora 33.
```

Comment:
blah, that's kinda confusing.  Should say "...as of Fedora 33 seccomp and sysctl changes"